edgeless_node/src/agent/mod.rs

```diff
@@ -18,7 +21,8 @@ enum AgentRequest {
     ),
     Patch(edgeless_api::common::PatchRequest),
     PatchResource(edgeless_api::common::PatchRequest, futures::channel::oneshot::Sender<anyhow::Result<()>>),
-    UPDATEPEERS(edgeless_api::node_management::UpdatePeersRequest),
+    UpdatePeers(edgeless_api::node_management::UpdatePeersRequest),
+    HealthStatus(futures::channel::oneshot::Sender<anyhow::Result<edgeless_api::node_management::HealthStatus>>),
```

Comment:
The naming between keepalive / health check is not consistent (health check is the better name).